tests/classes/CLI.js

```diff
@@ -27,9 +27,9 @@ describe('CLI', () => {
       expect(cli.loadedPlugins.length).to.equal(0);
     });
 
-    it('should set an empty inputArray when none is provided', () => {
+    it('should set an undefined inputArray when none is provided', () => {
```

Comment:
This test needs a tiny update once the variable setting from `undefined` to `null` is done.